src/domain/Score.java

```diff
@@ -1,4 +1,7 @@
 package domain;
 
 public class Score {
+	public static void score(int i, String string, int j) {
```

Comment:
Dit moet een constructor worden, geen static method.

De syntax voor een constructor is:
```java
public KlasseNaam(<parameters hier>) {
    <initializatie code hier>
}
```

Deze class heeft waarschijnlijk ook nog enkele attributen nodig (naam en score)